src/dotnet/APIView/apiview.yml

```diff
@@ -197,122 +197,141 @@ extends:
-          #         APIVIEW_BLOB__CONNECTIONSTRING: $(AzuriteConnectionString)
-          #         APIVIEW_COSMOS__CONNECTIONSTRING: $(CosmosEmulatorConnectionString)
-              
-          #     - script: |
-          #         npm install
-          #       workingDirectory: $(WebClientProjectDirectory)
-          #       displayName: "Install Client Dependencies"
-      
-          #     - script: |
-          #         npx playwright install --with-deps
-          #       workingDirectory: $(WebClientProjectDirectory)
-          #       displayName: "Install Playwright Browsers"
-      
-          #     - script: |
-          #         npx playwright test --project=unit-tests
-          #       workingDirectory: $(WebClientProjectDirectory)
-          #       displayName: "Run Client-Side Unit Tests"
-      
-          #     - task: PublishBuildArtifacts@1
-          #       inputs:
-          #         pathtoPublish: '$(Build.SourcesDirectory)\src\dotnet\APIView\APIViewWeb\Client\playwright-report'
-          #         artifactName: 'Client-Side Unit Test Reports'
-        
-          #     - ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal')) }}:
-          #       - template: /eng/pipelines/templates/steps/apiview-ui-tests.yml
-          #         parameters:
-          #           NodeVersion: $(NodeVersion)
-          #           WebClientProjectDirectory: $(WebClientProjectDirectory)
-          #           AzuriteConnectionString: $(AzuriteConnectionString)
-          #           CosmosEmulatorConnectionString: $(CosmosEmulatorConnectionString)
-      
-          #     - task: PublishTestResults@2
-          #       condition: succeededOrFailed()
-          #       inputs:
-          #         testResultsFiles: '**/*.trx'
-          #         testRunTitle: 'Tests against Windows .NET'
-          #         testResultsFormat: 'VSTest'
-          #         mergeTestResults: true
+          - job: 'Test'
+
+            pool:
+              name: azsdk-pool-mms-win-2022-general
+              vmImage: windows-2022
+
+            steps:
+              - script: |
+                  npm install -g @angular/cli@16.1.8
+                workingDirectory: $(SPAClientProjectDirectory)
+                displayName: "Install Angular CLI"
+
+              - script: |
+                  npm install
+                workingDirectory: $(WebClientProjectDirectory)
+                displayName: "Install npm packages"
+
+              - script: |
+                  npm install
+                workingDirectory: $(SPAClientProjectDirectory)
+                displayName: "Install Client SPA Packages"
+
+              - script: |
+                  ng test --no-watch --no-progress --browsers=ChromeHeadless
+                workingDirectory: $(SPAClientProjectDirectory)
+                displayName: "Run SPA Tests"
+              #- template: /eng/common/pipelines/templates/steps/cosmos-emulator.yml
```

Comment:
is the idea later to remove this code?

Reply:
Testing has been disabled for a while. I think it was during the review revisions restructure. II will reenable it as part of another change. But some of the tests are no longer required now.